pom.xml

```diff
@@ -217,6 +217,7 @@
 		    <module>bundles/org.eclipse.equinox.launcher.gtk.linux.aarch64</module>
 		    <module>bundles/org.eclipse.equinox.launcher.gtk.linux.ppc64le</module>
 		    <module>bundles/org.eclipse.equinox.launcher.gtk.linux.x86_64</module>
+		    <module>bundles/org.eclipse.equinox.launcher.gtk.linux.riscv64</module>
```

Comment:
We can get the sources but not enable the build by default until there is build machine available. So if you want drop this line for now and we can get the rest in.

Reply:
ok